samples/ChatApp.Telemetry/ChatApp.Server/ChatApp.Server.csproj

```diff
@@ -2,14 +2,16 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-    <LangVersion>7.3</LangVersion>
+    <TargetFramework>net5.0</TargetFramework>
+    <LangVersion>8.0</LangVersion>
```

Comment:
**mosikasite**: `9.0` or `latest`

`LangVersion` is no longer needed in this project.

Reply:
If I set `<Project Sdk="Microsoft.NET.Sdk.Web">` even TargetGramework is net5.0 but langversion detected as C# 7.3, this behavior is not detected with `Microsoft.NET.Sdk` though.

Reply:
set to use latest.